data/hotelchains.md

```diff
@@ -0,0 +1,356 @@
+# Hotel Chains. Not all might be accessible in test systems.
```

Comment:
Just a small suggestion as I found the sentence a bit confusing:
Not all might be accessible in test systems. -> 
Some of them might not be accessible in test environment.
or
Not all of them are accessible in test environment.

Reply:
updated